gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/discord/ListenerService.java

```diff
@@ -204,4 +205,21 @@ public String getTheQuestion(SlashCommandInteractionEvent event) {
         return question;
     }
 
+    public boolean IsUserExist(String author) throws GendoxException{
+        return userService.isUserExistByUserName(author);
+    }
```

Comment:
Remove this method, you can use directly the `userService.isUserExistByUserName(author)`.